app/views/mailers/mailshots_mailer/community_launch.text.erb

```diff
@@ -0,0 +1,57 @@
+Hi <%= @user.handle %>,
+
+It's been a year since we launched the new version of Exercism, and what an exciting year it's been! We've now helped over 1M people improve their coding skills, mentored over 300,000 people and gratefully received over $30,000 in donations. In the last few weeks, we've also launched new big features that we're really excited by!
+
+💬 The new Exercism Forum
+
+Our new forum is the place to share your programming journey with others, pick their brains when you get stuck, discussion new languages or concepts you discover, and to hang out with other similar-minded people. Come and say hello!
+
+Check the forum at https://forum.exercism.org
+
+——— 
+
+💡 Dig Deeper tabs
+
+Solving an exercise on Exercism is just the start.  The real magic comes afterwards - working with mentors, exploring others solutions, and now with our new Dig Deeper tab, exploring the different approaches and tradeoffs that people take when solving that exercise.
+
+Check out the blog post for more details: https://exercism.org/blog/dig-deeper
+
+——— 
+
+🤖 Deeper Automated Feedback
+
+Earlier this month we launched our biggest step in automation so far.  Our new Representation system mean that mentors' feedback will now be automatically given to see people who submit similar solutions in the future. You should now see more accurate automated feedback more frequently across a wider range of exercises.
+
+Check out the blog post for more details: https://exercism.org/blog/representers
+
+——— 
+
+👕 Swag Store
+
+Since Exercism started, people have been requesting swag. While it's finally here! T-shirts, mugs, mousemats, jigsaws and more. And we've priced everything at cost so as many people as possible can access it. So come get (geek-)stylish, with some Exercism swag 🙂
```

Comment:
```suggestion
Since Exercism started, people have been requesting swag. Well it's finally here! T-shirts, mugs, mousemats, jigsaws and more. And we've priced everything at cost so as many people as possible can access it. So come get (geek-)stylish, with some Exercism swag 🙂
```